plsql-frontend/src/test/java/org/sonar/plugins/plsqlopen/api/statements/CaseStatementTest.java

```diff
@@ -67,5 +67,16 @@ public void matchesCaseWithMemberIdentifier() {
     public void matchesLabeledCase() {
         assertThat(p).matches("<<foo>> case when x = 1 then foo := bar; end case foo;");
     }
+    
+    @Test
+    public void matchesCaseWithSelectorExpression() {
+        assertThat(p).matches("case foo + bar when 1 then foo := bar; end case;");
```

Comment:
@felipebz this test fails with OBJECT_REFERENCE. The same applies to the CASE_EXPRESSION.